src/server/server.ts

```diff
@@ -415,7 +415,7 @@ export class Server {
         let endpoint: Datex.Endpoint|undefined;
             
         // session/endpoint handling:
-        if ((this as any)._uix_init && Server.isBrowserClient(requestEvent.request) && (requestEvent.request.headers.get("Sec-Fetch-Dest") == "document" /*|| requestEvent.request.headers.get("Sec-Fetch-Dest") == "iframe"*/) && requestEvent.request.headers.get("connection")!="Upgrade") {
+        if ((this as any)._uix_init && Server.isBrowserClient(requestEvent.request) && (requestEvent.request.headers.get("Sec-Fetch-Dest") == "document" || requestEvent.request.headers.get("Uix-Inline-Backend") == "true" /*|| requestEvent.request.headers.get("Sec-Fetch-Dest") == "iframe"*/) && requestEvent.request.headers.get("connection")!="Upgrade") {
```

Comment:
Capitalization of `Uix-Inline-Backend` is inconsistent to https://github.com/unyt-org/uix/blob/dffafbe5dc7e3a78c39608108fe3c70fbfb7b850/src/routing/frontend-routing.ts#L247